wwwroot/index.html

```diff
@@ -156,211 +372,99 @@ <h2 class="development-title mt-2 mb-lg-5">Which we are proud of</h2>
-								Bitcoin's White Paper
+								<span class="highlight"> Wasabi 1.0 </span> released on the 10th anniversary of Bitcoin's White Paper
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line2">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" alt="Lightning Network integration" src="assets/interface.svg">
+							<img class="svg-icon mb-0" alt="Bitcoin.org wallet listing" title="Bitcoin.org wallet listing" src="assets/interface.svg">
 							<div class="card-body development-card-body">
-								Wasabi Wallet listed on <a class="bitcoin-anonymous" href="https://bitcoin.org/" target="_blank">
-									http://bitcoin.org </a>
+								Wasabi Wallet listed on <a class="bitcoin-anonymous" title="Bitcoin.org" href="https://bitcoin.org/" target="_blank">https://bitcoin.org </a>
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line3">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" alt="Lightning Network integration"
-								src="assets/miscellaneous.svg">
+							<img class="svg-icon mb-0" alt="Hardware Wallet Interface integration" title="Hardware Wallet Interface integration" src="assets/miscellaneous.svg">
 							<div class="card-body development-card-body">
 								<span class="highlight"> 1.1.4. Release: </span> Hardware Wallet Support
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line4">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" src="assets/seo-and-web.svg" alt="Support Android and iOS">
+							<img class="svg-icon mb-0" src="assets/seo-and-web.svg" alt="CoinJoin bounty" title="CoinJoin bounty">
 							<div class="card-body development-card-body">
 								Wasabi awarded with the CoinJoin Bounty
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line5">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" src="assets/busy.svg" alt="Set up a VPN service">
+							<img class="svg-icon mb-0" src="assets/busy.svg" alt="Largest CoinJoin transaction" title="Largest CoinJoin transaction">
 							<div class="card-body development-card-body">
 								Largest CoinJoin in History with 100 people
 							</div>
 						</div>
 					</div>
 					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line6">
 						<div class="card development-card h-100 px-2 py-3">
-							<img class="svg-icon mb-0" src="assets/file.svg" alt="JoinMarket integration">
+							<img class="svg-icon mb-0" src="assets/file.svg" alt="Wasabi downloads" title="Wasabi downloads">
 							<div class="card-body development-card-body">
-								Approximately 170,500 Wasabi downloads
+								Approximately 200,000 Wasabi downloads
 							</div>
 						</div>
 					</div>
 				</div>
 			</div>
 		</div>
-
```

Comment:
Our media appearances were considered important by @bharmat and @hgergoil. Are you OK to remove it? From my side, it makes no sense for this to be there like it is now, nobody will click on it.

Reply:
Agreed, we already discussed it with Riccardo to get rid of it.

Reply:
ACK